handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -450,4 +489,8 @@ static void releaseIfNeeded(ReferenceCounted counted) {
             ReferenceCountUtil.safeRelease(counted);
         }
     }
+
+    static boolean isTlsv13Supported() {
```

Comment:
Tls13 or TlsV13?

Reply:
I actually like `isTlsv13Supported` best... are you strong here ?

Reply:
Lowercase v is what Google's style guide says, if it matters.